Jenkinsfile

```diff
@@ -46,31 +49,9 @@ pipeline {
           }
           axis {
             name 'PLATFORM'
-            values 'linux/amd64', 'linux/arm64'
```

Comment:
Had to do this to be able to use the PLATFORM in the image tag which doesn't allow "/" as a character

Reply:
I meant PLATFORM != ARCH. 
you use docker arch in place of PLATFORM.

Reply:
The use of platform was deliberate. The intent is to have an aarch64 musl toolchain and an x86_64 musl toolchain available for each of linux/amd64 and linux/arm64 and have the

`${VARIANT_VERSION}-1-${RUST_VERSION}-aarch64` and 
`${VARIANT_VERSION}-1-${RUST_VERSION}-arm64` tags available as multiarch manifests so that Jenkins can, for example, cross compile from x86_64 to aarch64 musl without falling back to qemu

Reply:
I've renamed the confusing ARCH variable to CROSS_COMPILER_TARGET_ARCH and re-removed the duplicated base image building that was happening.